service/src/main/resources/config/aou/test/SC2023Q3R1/entity/t_zipcode_socioeconomic.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "t_zipcode_socioeconomic",
+  "idAttribute": "id",
+  "attributes": [
+    { "type": "SIMPLE", "name": "id", "dataType": "INT64" },
+    { "type": "SIMPLE", "name": "observation_datetime_utc", "dataType": "TIMESTAMP" },
+    { "type": "SIMPLE", "name": "observation_datetime_str", "dataType": "STRING" },
+    { "type": "SIMPLE", "name": "zip_code", "dataType": "STRING" },
+    { "type": "SIMPLE", "name": "assisted_income", "dataType": "DOUBLE" },
+    { "type": "SIMPLE", "name": "high_school_education", "dataType": "DOUBLE" },
+    { "type": "SIMPLE", "name": "median_income", "dataType": "DOUBLE" },
+    { "type": "SIMPLE", "name": "no_health_insurance", "dataType": "DOUBLE" },
+    { "type": "SIMPLE", "name": "poverty", "dataType": "DOUBLE" },
+    { "type": "SIMPLE", "name": "vacant_housing", "dataType": "DOUBLE" },
+    { "type": "SIMPLE", "name": "deprivation_index", "dataType": "DOUBLE" },
+    { "type": "SIMPLE", "name": "american_community_survey_year", "dataType": "INT64" }
+  ],
+
+  "sourceDataMapping": {
+    "dataPointer": "omop_dataset",
+    "tablePointer": { "rawSqlFile": "t_zipcode_socioeconomic.sql" },
+    "attributeMappings": {
+      "id": { "value": { "column": "person_id" } }
```

Comment:
I would expect this to be the unique id/primary key of the related table, not the `person` table. How many rows in `zip3_ses_map` and/or `observation` filtered on the zip concept id/strings are you expecting per `person`?

Reply:
Updated tables and SQL for t_zipcopde_socioeconomic